app/dashboard/notifications.py

```diff
@@ -406,117 +407,92 @@ def build_github_notification(bounty, event_name, profile_pairs=None):
+                            issue_comment = interest.issue_message
+                        )
+                )
+            context = {
+                'bounty': bounty,
+                'started_work': started_context}
+            msg['started'] = render_to_string(template_path + 'start_work.md', context)
 
+    elif event_name == 'stop_work':
+        started_work = [interest.profile for interest in bounty.interested.filter(pending=False).select_related('profile').all()]
+        started_work_handles = [profile.handle for profile in started_work]
+        context = {
+                    'bounty': bounty,
+                    'started_work': started_work,
+                    #stopped_work: [activity.profile for activity in bounty.activities.filter(activity_type='stop_work').all() if activity.profile not in started_work]
+                    'stopped_work': [activity.profile for activity in bounty.activities.filter(activity_type='stop_work').all() if activity.profile.handle not in started_work_handles]
+        }
+        msg = render_to_string(template_path + 'stop_work.md', context)
     elif event_name == 'work_submitted':
-        sub_msg = ""
-        if bounty.fulfillments.exists():
-            sub_msg = f"\n\n{bounty_owner if bounty_owner else 'If you are the bounty funder,'} " \
-                       "please take a look at the submitted work:\n"
-            for bf in bounty.fulfillments.all():
-                username = "@"+bf.fulfiller_github_username if bf.fulfiller_github_username else bf.fulfiller_address
-                link_to_work = f"[PR]({bf.fulfiller_github_url})" if bf.fulfiller_github_url else "(Link Not Provided)"
-                sub_msg += f"* {link_to_work} by {username}\n"
-
-        profiles = ""
-        if profile_pairs:
-            for i, profile in enumerate(profile_pairs, start=1):
-                profiles = profiles + f"\n {i}. [@{profile[0]}]({profile[1]})"
-            profiles += "\n\n"
-
-
-        msg = f"{status_header}__Work for {natural_value} {bounty.token_name} {usdt_value} has been submitted by__:\n" \
-              f"{profiles}{sub_msg}\n<hr>\n\n* Learn more [on the Gitcoin Issue Details page]({absolute_url})\n" \
-              "* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the " \
-              f"<a href='https://gitcoin.co/slack'>Gitcoin Slack</a>\n${amount_open_work} more funded " \
-              "OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)\n"
+            context = {
+                        'bounty': bounty,
+                        'fulfillments': [fulfillment.profile for fulfillment in bounty.fulfillments.select_related('profile').all()]
+                      }
+            msg = render_to_string(template_path + 'submit_work.md', context)
+
     elif event_name == 'work_done':
+        context = {
+                        'bounty': bounty
+                  }
```

Comment:
Can we fix the indentation throughout this file to conform to pep8?

Reply:
@mbeacom Done! Please let me know if you see anything oustanding